packages/account/src/Sections/Assessment/FinancialAssessment/financial-assessment.jsx

```diff
@@ -313,24 +313,24 @@ const FinancialAssessment = ({
         <React.Fragment>
             <Formik
                 initialValues={{
-                    income_source,
-                    employment_status,
-                    employment_industry,
-                    occupation,
-                    source_of_wealth,
-                    education_level,
-                    net_income,
-                    estimated_worth,
-                    account_turnover,
+                    income_source: '' || income_source,
```

Comment:
I would recommend using `??` operator rather than `||`.  With `??` if the value is only null or undefined, then it will move to fallback. With `||` if value is false or null or undefined, it will move to fallback